blockbot/blockbot.php

```diff
@@ -118,7 +119,7 @@ function blockbot_init_1(App $a) {
 		'WordPress/', 'http.rb/', 'Apache-HttpClient/', 'WordPress.com;', 'Pleroma',
 		'Dispatch/', 'Ruby', 'Java/', 'libwww-perl/', 'Mastodon/',
 		'lua-resty-http/', 'Tiny Tiny RSS/', 'Wget/', 'PostmanRuntime/',
-		'W3C_Validator/', 'NetNewsWire'];
+		'W3C_Validator/', 'NetNewsWire', 'FeedValidator/'];
```

Comment:
Are you sure blocking `FeedValidator` is a good idea? The w3.org (W3C) feed validator is a very useful tool to check if a RSS/Atom feed is valid and working. And as we are providing feeds it shouldn't be blocked, I think.

See:
https://validator.w3.org/feed/
https://validator.w3.org/services

Reply:
Errm. I only looked at the changes... Is it whitelisted or blacklisted. I'm not sure. :-)

Reply:
It is allow-listed.